packages/expo-crypto/android/src/main/java/expo/modules/crypto/CryptoModule.kt

```diff
@@ -0,0 +1,49 @@
+package expo.modules.crypto
+
+import android.content.Context
+import android.util.Base64
+
+import expo.modules.core.ExportedModule
+import expo.modules.core.ModuleRegistry
+import expo.modules.core.Promise
+import expo.modules.core.interfaces.ExpoMethod
+
+import java.security.MessageDigest
+import java.security.NoSuchAlgorithmException
+
+class CryptoModule(context: Context?) : ExportedModule(context) {
+  override fun onCreate(moduleRegistry: ModuleRegistry) { }
```

Comment:
Don't know enough about kotlin, but is this needed even if it's empty?

Reply:
Tbh, I kept this method only because it was kept in Java source. However this is not needed, because identical default implementation is provided in `RegistryLifecycleListener` interface which is implemented by `ExportedModule`.